### DISCUSSION TITLE
Import passwords, how?

### DISCUSSION THREAD

**Discussion by ghost** [Slion/Fulguris#464] [Q&A]
Specifically, I need to get my web passwords from my desktop (where I run Firefox) into my Android system, to use them with Fulguris. I have already exported them to a CSV file, that is no problem at all. But I can't find *any* Fulguris option about passwords at all, let alone about importing them from elsewhere.

I could stop here, and answers to the above question would be valuable. In reality there is an extra important constraint: there is no google account on the android system, so one cannot be used as a way station. That probably excludes any solution which relies on the anrdoid built in secrets store. Or does it?

-- 
Ian

**Comment by Slion** [comment]
You can't import password no. You can use Android autofill service though. Works well with Google Autofill. Other Autofill service should also work but from experience they are not so reliable. Hope that helps.
